package/MDAnalysis/lib/distances.py

```diff
@@ -61,29 +61,33 @@
 you wish to use is covered by distopia. For more information see the
 `distopia documentation`_.
 
-.. Table:: Functions available using the `distopia`_ backend.
-    :align: center
-
-    +-------------------------------------+-----------------------------------+
-    | Functions                           | Notes                             |
-    +=====================================+===================================+
-    | MDAnalysis.lib.distances.calc_bonds | Doesn't support triclinic boxes   |
-    +-------------------------------------+-----------------------------------+
+.. table:: Functions available using the `distopia`_ backend.
+   :align: center
+
+   +-----------------------------------------------+
+   | Functions                                     |
+   +===============================================+
+   | MDAnalysis.lib.distances.calc_bonds           |
+   +-----------------------------------------------+
+   | MDAnalysis.lib.distances.calc_angles          |
+   +-----------------------------------------------+
+   | MDAnalysis.lib.distances.calc_dihedrals       |
+   +-----------------------------------------------+
+   | MDAnalysis.lib.distances.distance_array       | 
+   +-----------------------------------------------+
+   | MDAnalysis.lib.distances.self_distance_array  |
```

Comment:
hooray – distance arrays!

Please also update the docs for the individual functions
- update available backends
- add a versionchanged